ci/script_multi.sh

```diff
@@ -32,8 +33,12 @@ elif [ "$COVERAGE" ]; then
 
 elif [ "$SLOW" ]; then
     TEST_ARGS="--only-slow --skip-network"
-    echo pytest -m "not single and slow" -v --durations=10 --junitxml=test-data-multiple.xml --strict $TEST_ARGS pandas
-    pytest      -m "not single and slow" -v --durations=10 --junitxml=test-data-multiple.xml --strict $TEST_ARGS pandas
+    # The `-m " and slow"` is redundant here, as `--only-slow` is already used (via $TEST_ARGS). But is needed, because with
+    # `--only-slow` fast tests are skipped, but each of them is printed in the log (which can be avoided with `-q`),
+    # and also added to `test-data-multiple.xml`, and then printed in the log in the call to `ci/print_skipped.py`.
+    # Printing them to the log makes the log exceed the maximum size allowed by Travis and makes the build fail.
+    echo pytest -n 2 -m "not single and slow" --durations=10 --junitxml=test-data-multiple.xml --strict $TEST_ARGS pandas
+    pytest      -n 2 -m "not single and slow" --durations=10 --junitxml=test-data-multiple.xml --strict $TEST_ARGS pandas
```

Comment:
not easy to see in the diff, but note that I added the `-n 2` here, so the slow tests are being called with two processes in `script_multi.sh` now.